run.sh

```diff
@@ -78,6 +91,9 @@ run_acq()
 	./tools/bl srccfg "$device" 5  1 0 "$oversample" 0 0 # 5.0V
 	./tools/bl srccfg "$device" 6  1 0 "$oversample" 0 0 # Temperature
 
+	# TODO: this chancfg table has to change to be similar to the one
+	# in run_cal, but I've no idea where do those shift values come
+	# from, so leave it for now.
```

Comment:
Basically, if you run a calibrate, it will tell you the shift and offset to use for a 16-bit mode run.  The values here are hard-coded based on an old calibration.

Normally you would use the `run_cal_acq` function you made which does the calibration first and automatically sends the correct channel shift and offset values for the subsequent acquisition.

Reply:
alright, thanks for clarifying